sharepa/analysis.py

```diff
@@ -37,7 +38,10 @@ def agg_to_two_dim_dataframe(agg):
         if dict not in [type(item) for item in bucket_as_dict.values()]:
             return bucket_to_dataframe('doc_count', agg.buckets)
         else:
-            name_of_lower_level = bucket_as_dict.keys()[0]
+            lower_level_dict = [item for item in bucket_as_dict.keys() if type(bucket_as_dict[item]) is dict]
+            if len(lower_level_dict) > 1:
+                raise ValueError('Two dimensional data can only convert a 2 level aggregation (with 1 aggregation at each level)')
```

Comment:
Could you add a test that causes this error to be raised? You can assert that certain errors are thrown with the assert_raises pytest function:
```python
import pytest

def test_zero_division():
    with pytest.raises(ZeroDivisionError):
        1 / 0
```